From ca1fdc8b251897266839a27a38980c28f3a5b5e6 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <perex@perex.cz>
Date: Wed, 19 Aug 2020 16:04:48 +0200
Subject: [PATCH 46/51] Revert "amd/renoir/acp: use the machine driver's name
 for top-level's conf"

This reverts commit 35277f419b1dc0f8500d3ab4bceabf6cad05c76d.

We use the snd_acp3x_rn kernel module name again with the correct
sysfs lookup.

Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 ucm2/module/acp_pdm_mach.conf | 1 -
 ucm2/module/snd_acp3x_rn.conf | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)
 delete mode 120000 ucm2/module/acp_pdm_mach.conf
 create mode 120000 ucm2/module/snd_acp3x_rn.conf

diff --git a/ucm2/module/acp_pdm_mach.conf b/ucm2/module/acp_pdm_mach.conf
deleted file mode 120000
index b56794a7bc1f..000000000000
--- a/ucm2/module/acp_pdm_mach.conf
+++ /dev/null
@@ -1 +0,0 @@
-lib/linked.conf
\ No newline at end of file
diff --git a/ucm2/module/snd_acp3x_rn.conf b/ucm2/module/snd_acp3x_rn.conf
new file mode 120000
index 000000000000..b56794a7bc1f
--- /dev/null
+++ b/ucm2/module/snd_acp3x_rn.conf
@@ -0,0 +1 @@
+lib/linked.conf
\ No newline at end of file
-- 
2.16.4

